src/yoroi-wallets/cardano/api/api.ts

```diff
@@ -89,9 +89,34 @@ export const getPoolInfo = (request: PoolInfoRequest, config: BackendConfig): Pr
 }
 
 export const getNFTs = async (assets: RemoteAsset[], config: BackendConfig): Promise<YoroiNft[]> => {
-  const request = {assets: assets.map((asset) => ({nameHex: asset.name, policy: asset.policyId}))}
-  const response = await fetchDefault('multiAsset/metadata', request, config)
-  return parseNFTs(response, config.NFT_STORAGE_URL)
+  const assetIds = assets.map((asset) => asset.assetId)
+  const metadataPayload = {assets: assets.map((asset) => ({nameHex: asset.name, policy: asset.policyId}))}
+  const [metadataResponse, supplies] = await Promise.all([
```

Comment:
what are the types of metadataResponse and supplies?
dont use "response" in the name of a variable